frontend/src/scss/_uswds-theme-general.scss

```diff
@@ -25,7 +25,7 @@ Relative image file path
 ----------------------------------------
 */
 
-$theme-image-path: "../img";
+$theme-image-path: "../../node_modules/uswds/dist/img";
```

Comment:
⚠️ do you foresee any issues with linking to the uswds node module images? if not, should we be referencing the images in `dist` or `src` or does it not matter..?

Reply:
the path of these images will not change on USWDS version upgrades unless it's a major change (potentially) which USWDS doesn't do very often. They're only on version 2 in its 5+ years of being around.

Reply:
USWDS references `dist/img` in https://designsystem.digital.gov/documentation/developers/#where-things-live but i'd guess it doesn't make a difference. i can ask, tho.